.github/workflows/build.yaml

```diff
@@ -100,3 +100,16 @@ jobs:
         NUM_NODES: 0
         WITHOUT_ENVSUBST: 1
         EXTRA_GMSA_DEPLOY_ARGS: --tolerate-master
+  chart-integration:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
```

Comment:
can you try bumping this to 1.17.

I've seen the same errors you are seeing in CI job when trying to build w/ go 1.16 when deps moved to 1.17.

```
Error: /home/runner/go/pkg/mod/sigs.k8s.io/json@v0.0.0-20211020170558-c049b76a60c6/internal/golang/encoding/json/encode.go:1249:12: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
Error: /home/runner/go/pkg/mod/sigs.k8s.io/json@v0.0.0-20211020170558-c049b76a60c6/internal/golang/encoding/json/encode.go:1255:18: sf.IsExported undefined (type reflect.StructField has no field or method IsExported)
```

Plus all the other jobs in this file are using 1.17 :)

Reply:
@marosset just needed that extra set of eyes. Thanks.